.github/workflows/unit-tests.yml

```diff
@@ -202,13 +206,14 @@ jobs:
           -I ${{github.workspace}}/rdkservices/Tests/headers/rdk/iarmbus
           -I ${{github.workspace}}/rdkservices/Tests/headers/rdk/iarmmgrs-hal
           -I ${{github.workspace}}/rdkservices/Tests/headers/ccec/drivers
+          -I ${{github.workspace}}/rdkservices/Tests/mocks
```

Comment:
please remove

as you added folder headers/rdkshell to generated you should include that folder instead